packages/react-native-gesture-handler/src/web/tools/GestureHandlerWebDelegate.ts

```diff
@@ -52,7 +53,11 @@ export class GestureHandlerWebDelegate
     this.setTouchAction();
     this.setContextMenu();
 
-    this.eventManagers.push(new PointerEventManager(this.view));
+    const shouldSendHoverEvents = handler instanceof HoverGestureHandler;
+
+    this.eventManagers.push(
+      new PointerEventManager(this.view, shouldSendHoverEvents)
+    );
```

Comment:
Why the unnecessary const? I think if you want to clarify what it does it is better to use a comment
```suggestion
    this.eventManagers.push(
      new PointerEventManager(this.view, handler instanceof HoverGestureHandler)
    );
```